eth/stagedsync/stagedsynctest/harness.go

```diff
@@ -64,24 +64,7 @@ func InitHarness(ctx context.Context, t *testing.T, cfg HarnessCfg) Harness {
 		false,
 		nil,
 	)
-	stateSyncStages := stagedsync.DefaultStages(
-		ctx,
-		stagedsync.SnapshotsCfg{},
-		stagedsync.HeadersCfg{},
-		bhCfg,
-		stagedsync.BlockHashesCfg{},
-		stagedsync.BodiesCfg{},
-		stagedsync.SendersCfg{},
-		stagedsync.ExecuteBlockCfg{},
-		stagedsync.HashStateCfg{},
-		stagedsync.TrieCfg{},
-		stagedsync.HistoryCfg{},
-		stagedsync.LogIndexCfg{},
-		stagedsync.CallTracesCfg{},
-		stagedsync.TxLookupCfg{},
-		stagedsync.FinishCfg{},
-		true,
-	)
+	stateSyncStages := stagedsync.DefaultStages(ctx, stagedsync.SnapshotsCfg{}, stagedsync.HeadersCfg{}, bhCfg, stagedsync.BlockHashesCfg{}, stagedsync.BodiesCfg{}, stagedsync.SendersCfg{}, stagedsync.ExecuteBlockCfg{}, stagedsync.HistoryCfg{}, stagedsync.LogIndexCfg{}, stagedsync.CallTracesCfg{}, stagedsync.TxLookupCfg{}, stagedsync.FinishCfg{}, true)
```

Comment:
maybe to the same codestyle as above? It would be easier to read..

Reply:
done